Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/DocumentPropertyEditor/PrefabAdapter.cpp

```diff
@@ -44,7 +44,6 @@ namespace AzToolsFramework::Prefab
         if (m_prefabOverridePublicInterface->AreOverridesPresent(entityId, relativePathFromEntity.ToString()))
         {
             adapterBuilder->BeginPropertyEditor<PrefabPropertyEditorNodes::PrefabOverrideProperty>();
-            adapterBuilder->Attribute(AZ::DocumentPropertyEditor::Nodes::PropertyEditor::SharePriorColumn, true);
```

Comment:
Removed this as it was throwing errors if the icon is the first thing put in a row. This can be added at a later point if we want this to come after a certain property/label